helpers/channels/deleteChannel.ts

```diff
@@ -1,9 +1,8 @@
 import type { Bot } from "../../bot.ts";
-import { DiscordChannel } from "../../types/discord.ts";
 
 /** Delete a channel in your server. Bot needs MANAGE_CHANNEL permissions in the server. Bot needs MANAGE_THREADS permissions in the server if deleting thread. */
-export async function deleteChannel(bot: Bot, channelId: bigint, reason?: string) {
-  await bot.rest.runMethod<DiscordChannel>(
+export async function deleteChannel(bot: Bot, channelId: bigint, reason?: string): Promise<void> {
+  return void await bot.rest.runMethod(
```

Comment:
What is the reasoning to add return void here?

Reply:
It is a personal preference. The code works just fine without it, but I feel that it makes the code more consistent since a value is always returned, even if its void.

Reply:
In that case id prefer return await bot.rest.runMethod<void>()

Reply:
Sure, that works too. Going to update.